pkg/action/cobra.go

```diff
@@ -66,6 +74,32 @@ func CobraImpl(a *Action, streams cli.Streams) (*cobra.Command, error) {
 	return cmd, nil
 }
 
+func processOpts(opts TypeOpts, schema OptionsList, processors map[string]launchr.ValueProcessor) error {
```

Comment:
This must be done inside `SetInput`, because the behavior must be the same for any execution environment.

Reply:
For me, arguments must have similar behavior. Should we add the same property for args?